mscore/scoreview.cpp

```diff
@@ -141,7 +141,19 @@ ScoreView::ScoreView(QWidget* parent)
       double mag  = preferences.getDouble(PREF_SCORE_MAGNIFICATION) * (mscore->physicalDotsPerInch() / DPI);
       _matrix     = QTransform(mag, 0.0, 0.0, mag, 0.0, 0.0);
       imatrix     = _matrix.inverted();
-      _magIdx     = preferences.getDouble(PREF_SCORE_MAGNIFICATION) == 1.0 ? MagIdx::MAG_100 : MagIdx::MAG_FREE;
+      switch (preferences.getInt(PREF_SCORE_ZOOM_TYPE)) {
+            case 0: // Percentage
```

Comment:
why is there no `case 1`?

Reply:
oh, is `default` case 1?

Reply:
Yeah. Figured since the "rule" is usually to provide a default with switches, might as well double-down with it and 1 as being equivalent.

Reply:
I'd prefer if you were explicit:

    case 1: // Page Width (set as default within prefsdialog.ui)
          Q_FALLTHROUGH();
    default:

Reply:
'Twas how it was before rebasing, but figured I'd save an endangered line or two since for now there are no other available options for zoom options and it's "nice" to have default laying around down at the bottom.

Reply:
The `default` case does need to be at the bottom, and `case 1` right before it.

Or, if you want them to appear in numerical order, you can just omit `default`. But regardless, you really shouldn't omit `case 1`.

Reply:
Why not?

Reply:
Bacause it is confusing